### ISSUE TITLE
[core/useScorll] arrivedState for noscroll element should have some meaningful value

### ISSUE THREAD

**Issue by machao7** [vueuse/vueuse#3337]
### Describe the bug

arrivedState' right and bottom should be true for noscroll element ?

https://codesandbox.io/s/reverent-hellman-knctcy?file=/src/App.vue:379-391

### Reproduction

https://codesandbox.io/s/reverent-hellman-knctcy?file=/src/App.vue:379-391

### System Info

```Shell
System:
    OS: Windows 10 10.0.22621
    CPU: (8) x64 Intel(R) Core(TM) i7-9700 CPU @ 3.00GHz
    Memory: 13.26 GB / 31.79 GB
  Binaries:
    Node: 18.16.1 - C:\Program Files\nodejs\node.EXE
    npm: 6.14.18 - ~\AppData\Roaming\npm\npm.CMD
    pnpm: 7.14.1 - ~\AppData\Roaming\npm\pnpm.CMD
  Browsers:
    Chrome: 116.0.5845.97
    Edge: Spartan (44.22621.2134.0), Chromium (115.0.1901.203)
    Internet Explorer: 11.0.22621.1
```

### Used Package Manager

npm

### Validations

- [X] Follow our [Code of Conduct](https://github.com/vueuse/vueuse/blob/main/CODE_OF_CONDUCT.md)
- [X] Read the [Contributing Guidelines](https://github.com/vueuse/vueuse/blob/main/CONTRIBUTING.md).
- [X] Read the [docs](https://vueuse.org/guide).
- [X] Check that there isn't [already an issue](https://github.com/vueuse/vueuse/issues) that reports the same bug to avoid creating a duplicate.
- [X] Make sure this is a VueUse issue and not a framework-specific issue. For example, if it's a Vue SFC related bug, it should likely be reported to https://github.com/vuejs/core instead.
- [X] Check that this is a concrete bug. For Q&A open a [GitHub Discussion](https://github.com/vueuse/vueuse/discussions).
- [X] The provided reproduction is a [minimal reproducible example](https://stackoverflow.com/help/minimal-reproducible-example) of the bug.

**Comment by 1001v** [comment]
I agree. When there are no scroll, you can say it has arrived to bottom as well as to top.

**Comment by bblanchon** [comment]
I also think it makes sense for `top` and `right` to be `true` for no scroll elements.
Could you reopen this issue?
